spec/models/video_rspec.rb

```diff
@@ -4,4 +4,32 @@
   it { should validate_presence_of(:title) }
   it { should validate_presence_of(:description) }
   it { should belong_to(:category) }
+
+  describe "search_by_title" do

```

Comment:
I would recommend to stick to this convention "http://www.betterspecs.org/#describe" when you describe methods for specs. `.` for class method and `#` for instance method. Makes it clear from start what is going on.